AnnouncementAddViewController.m

```diff
@@ -109,6 +109,10 @@ - (IBAction)saveAnnouncement:(id)sender {
     if (self.announcementTitle.text.length == 0 || self.announcementContent.text.length == 0 || [self.announcementContent.text isEqualToString:@"Context"]) {
         [self displayAlertWithTitle:@"Error" andContext:@"Please fill all the information"];
     } else {
+        if (self.announcementTitle.text.length > 15) {
```

Comment:
It is better practice to make this as `else if` instead of putting it under `else`.